scripts/buildkite/release/push-release.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+set -euox pipefail
+
+NEW_GIT_TAG=$(buildkite-agent meta-data get release-version)
+
+if [ "$RELEASE" == "false" ]; then
+    TAG=nightly
+    title="Nightly $NEW_GIT_TAG"
+else
+    TAG=$NEW_GIT_TAG
+    title="Release $TAG"
+fi
+
+buildkite-agent artifact download artifacts/changes.md .
+# shellcheck disable=SC2155
+export CHANGES=$(cat artifacts/changes.md)
+
+# shellcheck disable=SC2155
+export NODE_TAG=$(buildkite-agent meta-data get node-tag)
+
+# shellcheck disable=SC2034
+export BUMP_CHANGES=xxx
+
+# shellcheck disable=SC2034
+export DOCKER_SHA=xxx
+
+# shellcheck disable=SC2016
+envsubst \
```

Comment:
I don't have `envsubst` in scope in my typical shell, but maybe its brought into scope by `./scripts/buildkite/release/flake.nix` — could you check again that this is indeed the case? 🤔